iris_gpubench/nvidia_nsight/setup_nsights.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+# Set up system
+cat /proc/sys/kernel/perf_event_paranoid
+
+sudo sh -c 'echo 2 >/proc/sys/kernel/perf_event_paranoid'
+
+sudo sh -c 'echo kernel.perf_event_paranoid=2 > /etc/sysctl.d/local.conf'
+
+# Install Nsights Package Manager for Ubuntu
+apt update
+apt install -y --no-install-recommends gnupg
+echo "deb http://developer.download.nvidia.com/devtools/repos/ubuntu$(source /etc/lsb-release; echo "$DISTRIB_RELEASE" | tr -d .)/$(dpkg --print-architecture) /" | tee /etc/apt/sources.list.d/nvidia-devtools.list
+apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
+apt update
+apt install nsight-systems-cli
```

Comment:
These should be prefixed with `sudo` and the script should be ran without sudo so it's only the steps which require it running as root

Reply:
For testing if you try to run these scripts against a sacrificial VM with a fed ID it should work with `.../setup_nsights.sh`